documentation/Arc42.md

```diff
@@ -179,10 +179,11 @@ There is a web interface between frontend and backend.<br /><br />
 
 Detailed API specs available under:
 
-[https://github.com/catenax-ng/product-dft-backend/blob/Tractus-X_Eclipse_Repo_Changes_0.4/src/main/resources/open-api/sde-backend-openapi.yaml](https://github.com/catenax-ng/product-dft-backend/blob/Tractus-X_Eclipse_Repo_Changes_0.4/src/main/resources/open-api/sde-backend-openapi.yaml)
+[https://github.com/catenax-ng/tx-dft-backend/blob/main/modules/sde-core/src/main/resources/sde-open-api.yml](https://github.com/catenax-ng/tx-dft-backend/blob/main/modules/sde-core/src/main/resources/sde-open-api.yml)
```

Comment:
Make sure to move this to eclipse-tractusx

Reply:
We have updated changes in file.
documentation/Arc42.md